components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -6476,7 +6480,7 @@ def reexportedName(name):
         if haveLegacyWindowAliases:
             cgThings.append(
                 CGList([CGGeneric("const legacy_window_aliases: &[&[u8]] = &["),
-                        CGIndenter(CGList([CGGeneric(str_to_const_array(name)) for
+                        CGIndenter(CGList([CGGeneric(str_to_byte_slice(name)) for
```

Comment:
I think we need to change `&[&[u8]]` -> `&[&CStr]`, although this might require more changes.